pkg/policy/rule.go

```diff
@@ -180,6 +180,19 @@ func (r *rule) canReach(ctx *SearchContext, state *traceState) api.Decision {
 	// separate loop is needed as failure to meet FromRequires always takes
 	// precedence over FromEndpoints
 	for _, r := range r.Ingress {
+		for _, entity := range r.FromEntities {
+			l, ok := api.EntitySelectorMapping[entity]
```

Comment:
Can you do this validation in `func (r *rule) validate() error {`?

This would let it fail at import time which will give direct feedback to the user.

Reply:
done